Makefile

```diff
@@ -1,10 +1,13 @@
 all: patch-swagger-doc format
 
-buf-gen:
+buf-gen: init-hooks
```

Comment:
Tracking a hook in a repo requires some finagling, this is the lowest-impact way I could think of accomplishing it. We will have to run `buf-gen`, so this change makes `init-hooks` a dependency of the `buf-gen`